src/server.js

```diff
@@ -6,13 +6,15 @@ const dnsServer = dns.createServer();
 const https = require('https');
 const types = {
   1: 'A',
+  5: 'CNAME',
   16: 'TXT'
 };
 const API_BASE = ['', 'v1', 'dns'];
 const RECORD_TTL = 600;
 let records = {
   TXT: {},
-  A: {}
+  A: {},
+  CNAME: {},
```

Comment:
IIUC with this change we store CNAME values, but we don't give them back on client requests, right? Should we add them to https://github.com/fxbox/dns-server/pull/8/files#diff-e6a5b42b2f7a26c840607370aed5301aR31